keyvault/client/src/main/java/com/azure/keyvault/SecretRequestAttributes.java

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.keyvault;
+
+import com.azure.keyvault.models.SecretAttributes;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+
+/**
+ * The object attributes managed by the Secret service.
+ */
+class SecretRequestAttributes {
+
+    /**
+     * Creates an instance of SecretRequestAttributes. Reads secretAttributes.notBefore, secretAttributes.expires and secretAttributes.enabled fields
+     * from {@code secretAttributes}
+     * @param secretAttributes the {@link SecretAttributes} object with populated attributes
+     */
+    SecretRequestAttributes(SecretAttributes secretAttributes) {
+        if (secretAttributes.notBefore() != null) {
+            this.notBefore = secretAttributes.notBefore().toEpochSecond();
+        }
+        if (secretAttributes.expires() != null) {
+            this.expires = secretAttributes.expires().toEpochSecond();
+        }
+        this.enabled = secretAttributes.enabled();
+    }
+
+    /**
+     * The secret value.
+     */
+    @JsonProperty(value = "value")
```

Comment:
Using `@JsonProperty()` is enough if the field name is the same as the JSON key name

Reply:
I agree, but this is one of those cases where I actually like being more verbose - it makes it less fragile if some architect decides to rename field names thinking it is inconsequential. @conniey I would recommend that you do this also.